ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/CCDCase.java

```diff
@@ -26,4 +27,6 @@
     private String defendantEmail;
     private CCDResponse response;
     private String respondedAt;
+    private CCDSettlement settlement;
+    private String settlementReachedAt;
```

Comment:
Can we change the type to `LocalDateTime` here?

Reply:
LocalDateTime is not supported at ccd at the moment, https://tools.hmcts.net/confluence/display/RCCD/CCD+Supported+Field+Types

Reply:
done